gradle.properties

```diff
@@ -1,7 +1,7 @@
 # JUnit5 jupiter uses version 5.5.2 and JUnit Platform uses version 1.5.2
```

Comment:
What about the comment here?

Reply:
Good that you spotted that. This was left over from before using the JUnit 5 BOM and should've been removed when we used that instead of creating Jupiter and Platform version by string concatenation.